### DISCUSSION TITLE
Vanjaro for DNN as an Extension

### DISCUSSION THREAD

**Discussion by 21Junipers** [vanjarosoftware/Vanjaro.Platform#1691] [Q&A] [answered]
*Found it* 

Could someone point me to current documentation of adding Vanjaro as an extension? The pre-release process is of course no longer valid and due to wording on the front page ("also available as an extension") it is unclear to me if there is a separate version for this process.

**Comment by Tychodewaard** [comment]
Installing an extension is pretty much the same.
https://www.youtube.com/watch?v=PjoNRRsOc6w

**Reply by 21Junipers** [reply]
Thanks and my apologies to you (and Mandeeps below). My question was not stated clearly. From the video Mandeeps links below, I understood the installation was the same as any other extension. But I was not finding the "Vanjaro for DNN" extension. I simply missed it in releases [here](https://github.com/vanjarosoftware/Vanjaro.Platform/releases).  Still getting used to how Github organizes things.

I understand why "Vanjaro for DNN" is named that way. Vanjaro is indeed Vanjaro and not DNN.  The more established Vanjaro becomes, the more obvious that naming convention will be to users.

P.S. Probably not the place for it, but I learned yesterday that indeed "git" was slang. Thanks BritBox. Interesting in this context!

**Comment by meetmandeep** [comment, marked as answer]
For anyone else looking: https://www.youtube.com/watch?v=CjBhaWPvxh8

**Comment by mpongan** [comment]
Is this still available? I can't find the link to download the extension anymore. :(

**Reply by mpongan** [reply]
I found it. No worries.

**Comment by mpongan** [comment]
I installed Vanjaro (1.5) on our Test DNN Site with 9.12 version.
It installed successfully and tried creating a new page based on the video.

It shows the Blank Page with no option to pick a template. The menu seems like disabled too.

![image](https://github.com/vanjarosoftware/Vanjaro.Platform/assets/91706738/14efbe97-3557-4af8-a836-c1a5d849a348)

I see some console errors as well if it helps.

![image](https://github.com/vanjarosoftware/Vanjaro.Platform/assets/91706738/4dc89d2e-28d5-4ef0-8d29-fa24b00bb418)
